### PR TITLE
Migrate Automatic HTTPS rewrites and remove bold from definition in p…

### PR DESCRIPTION
**Pull request by angelampcosta**
…artials

### CODE REVIEW

File: products/ssl/src/content/_partials/_automatic-https-rewrites-definition.md [hunk 1]
```diff
@@ -1 +1 @@
-**Automatic HTTPS Rewrites** prevents end users from seeing "mixed content" errors by rewriting URLs from `http` to `https` for resources or links on your web site that can be served with HTTPS.
\ No newline at end of file
+Automatic HTTPS Rewrites prevents end users from seeing "mixed content" errors by rewriting URLs from `http` to `https` for resources or links on your web site that can be served with HTTPS.
```
Comment (kodster28) [inline products/ssl/src/content/_partials/_automatic-https-rewrites-definition.md:1]:
```suggestion
Automatic HTTPS Rewrites prevent end users from seeing "mixed content" errors by rewriting URLs from `http` to `https` for resources or links on your web site that can be served with HTTPS.
```

File: products/ssl/src/content/edge-certificates/additional-options/automatic-https-rewrites.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+<AutomaticHTTPSRewritesDefinition/>
+
+## How to use Automatic HTTPS Rewrites
+
+If your site contains links or references to HTTP URLs that are also available securely via HTTPS, Automatic HTTPS Rewrites can help. If you connect to your site over HTTPS and the lock icon is not present, or has a yellow warning triangle on it, your site may contain references to HTTP assets (“mixed content”).
+
+Mixed content is often due to factors not under the website owner’s control such as embedded third-party content or complex content management systems. By rewriting URLs from “http” to “https”, Automatic HTTPS Rewrites simplifies the task of making your entire website available over HTTPS, helping to eliminate mixed content errors and ensuring that all data loaded by your website is protected from eavesdropping and tampering.
+
+Enable **Automatic HTTPS Rewrites** via the **SSL/TLS** app to rewrite HTTP resources as HTTPS at Cloudflare without making any changes to the HTML source at your origin server.
```
Comment (kodster28) [inline products/ssl/src/content/edge-certificates/additional-options/automatic-https-rewrites.md]: I'd separate this out into another section, maybe change this one to `Additional details` and make the how to `Enable Automatic HTTPS Rewrites`.

File: products/ssl/src/content/edge-certificates/additional-options/automatic-https-rewrites.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+## Limitations
+
+The HTTP resources are checked to ensure they are accessible via HTTPS before a rewrite is applied. In cases where a resource is not available over HTTPS, Cloudflare cannot rewrite the URL.
```
Comment (kodster28) [inline products/ssl/src/content/edge-certificates/additional-options/automatic-https-rewrites.md]: I'd consolidate this into one sentence and remove the passive voice if possible.

File: products/ssl/src/content/edge-certificates/additional-options/automatic-https-rewrites.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Some resources are loaded by JavaScript or CSS via HTTP when the site is loaded in a browser. You will see mixed content warnings in those situations. To determine which URLs do not have HTTPS support, Cloudflare uses data from EFF’s HTTPS Everywhere and Chrome’s HSTS preload list. If your zone is not on one of these lists, only active content will be rewritten. Passive content (such as images) will not be rewritten and will still cause mixed content errors.
+
+If a third-party domain supports HTTPS and is not rewritten automatically, you can manually change those links to relative links or HTTPS links.  Alternatively, you can ask the third-party domain owner to submit their site for inclusion in the HTTPS Everywhere rulesets, which [accept pull requests on GitHub](https://github.com/EFForg/https-everywhere/).
+
+## Related Resources
```
Comment (kodster28) [inline products/ssl/src/content/edge-certificates/additional-options/automatic-https-rewrites.md]: I'd move these related resources into the body of the description and remove this section entirely, if possible.

**Comment by kodster28** [comment]
Oh, also don't forget to update the main `Index` page with the new link to dev docs (vs the current support link).

**Review by kodster28 — CHANGES_REQUESTED** [review]

**Review by kodster28 — APPROVED** [review]

**Comment by Soeparmadi** [comment]
How are you?

**Pull request merged**